rgd/utility.py

```diff
@@ -23,3 +24,11 @@ def _field_file_to_local_path(field_file: FieldFile) -> Generator[Path, None, No
                 yield Path(dest_stream.name)
         else:
             yield Path(file_obj.name)
+
+
+def compute_checksum(file_path, chunk_num_blocks=128):
+    sha256 = hashlib.sha256()
```

Comment:
In Girder, we often use sha512.  Perhaps we should here for consistency?

Reply:
I have no opinion/experience on what to use. I chose sha256 over md5 because:

> Unless otherwise noted, sha256 is the RECOMMENDED algorithm for checksums in
JOIA.

From https://gitlab.kitware.com/vigilant/joia/-/blob/master/joia/docs/schema/Overview.md#conventions

Reply:
Then sha256 makes sense.